spec/migrations/20180618084757_add_backup_zone_id_to_ext_management_system.rb

```diff
@@ -0,0 +1,5 @@
+class AddBackupZoneIdToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :ext_management_systems, :backup_zone, :type => :bigint, :index => true
```

Comment:
I know this was here from the original PR, but I'm not a fan of the name of this column.

`backup` makes it sound like it will get used if there is a problem rather than as a part of normal operation (i.e. resume).

I haven't been able to come up with anything I'm really happy with, but here are some of my thoughts:

- `runtime_zone`
- `zone_before_pause`

Any other ideas? @agrare @Fryguy ?

Reply:
`original_zone`?

Reply:
I also considered `original_zone` or `last_zone`, but really this is something specific to pausing an ems so I would rather make that obvious from the attribute name.

Spoke about it for a bit with @jrafanie, our "Master Namer of Things" and we came up with `zone_when_paused`. How about that?

Reply:
Naming is hard

Reply:
Don't want to bikeshed too much about the name, but `zone_when_paused` would be the maintenance zone, I'd think `zone_before_pause` would be better than `zone_when_paused`